src/components/PageHarvest.js

```diff
@@ -28,6 +28,7 @@ class PageHarvest extends Component {
 
   harvestHandler(spec) {
     const { dispatch, token, queue } = this.props
+    dispatch(uiNotificationNew({ type: 'info', message: 'Harvesting started.', timeout: 5000 }))
```

Comment:
what is this timeout? vs the AlertList timeout (which is shorter)

Reply:
The timeout specifies when it should auto-hide.  I've removed the AlertList default.